docs/source/tutorials/base/how_to_train/classification.rst

```diff
@@ -1,7 +1,234 @@
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/algorithms/classification/multi_class_classification>`.
+
+  We also can modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (otx) ...$ otx find --task classification
+
+  +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+  |      TASK      |                         ID                        |          NAME         |                                        PATH                                       |
+  +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+  | CLASSIFICATION | Custom_Image_Classification_MobileNet-V3-large-1x | MobileNet-V3-large-1x | otx/algorithms/classification/configs/mobilenet_v3_large_1_cls_incr/template.yaml |
+  | CLASSIFICATION |    Custom_Image_Classification_EfficinetNet-B0    |    EfficientNet-B0    |    otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template.yaml   |
+  | CLASSIFICATION |   Custom_Image_Classification_EfficientNet-V2-S   |   EfficientNet-V2-S   |   otx/algorithms/classification/configs/efficientnet_v2_s_cls_incr/template.yaml  |
+  +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+
+To have a specific example in this tutorial, all commands will be run on the :ref:`MobileNet-V3-large-1x <classificaiton_models>`  model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2.  Next, we need to create train/validation sets. OTX supports auto-split functionality for the multi-class classificaiton. For other classification types we need to prepare splits in advance
+Let's prepare an OTX classification workspase running the following command:
+
+.. note::
+
+  Currently, OTX supports auto-split only for multi-class classificaiton. For the multi-label and hierarchical tasks we need to prepare data splits in advance.
+
+.. code-block::
+
+  (otx) ...$ otx build --train-data-roots data/flower_photos --model MobileNet-V3-large-1x
+
+  [*] Load Model Template ID: Custom_Image_Classification_MobileNet-V3-large-1x
+  [*] Load Model Name: MobileNet-V3-large-1x
+  [*] Saving data configuration file to: ./otx-workspace-CLASSIFICATION-MobileNet-V3-large-1x/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-CLASSIFICATION-MobileNet-V3-large-1x
+
+It will create **otx-workspace-CLASSIFICATION** with all necessery configs for MobileNet-V3-large-1x, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset
```

Comment:
```suggestion
It will create **otx-workspace-CLASSIFICATION** with all necessery configs for MobileNet-V3-large-1x, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
```